composer.json

```diff
@@ -22,7 +22,9 @@
     "require": {
         "php": "^7.3",
         "doctrine/common": "^2.3",
+        "psr/cache": "^1.0",
         "sonata-project/cache": "^1.0 || ^2.0",
+        "sonata-project/core-bundle": "^3.15.1",
```

Comment:
You should remove this dependency. That’s the reason why we must release new major versions for each bundle

Reply:
My bad :man_facepalming: 
Thank you!